src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,15 +41,20 @@ public class RobotContainer {
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
+    motor3.setInverted(true);
+    motor4.setInverted(true);
+    motor4.follow(motor3);
+    motor2.follow(motor1);
     // Configure the trigger bindings
     configureBindings();
   }
   public void teleopPeriodic() {
-    double throttle = m_driverController.getLeftY();
-    double turn = m_driverController.getRightX();
-    System.out.println("throttle is: " + throttle);
-    System.out.println("turn is: " + turn);
-    motor1.set(throttle);
+    // red xbox joysticks are inverted
+    double throttle = -m_driverController.getLeftY()*.75;
```

Comment:
I have two requests:

1.  Add spaces before and after the multiplication operator
2.  Move the number value to a constant (see "PI" here: https://www.learncs.online/lessons/java/static)

This feedback also applies to the next line.